payment-intent/python/payment-intent-auto.py

```diff
@@ -0,0 +1,22 @@
+import http.client
+import json
+
+conn = http.client.HTTPSConnection("api.dojo.tech")
+payload = json.dumps({
+  "amount": {
+    "value": 1000,
+    "currencyCode": "GBP"
+  },
+  "reference": "Order 234",
+  "captureMode": "Auto"
+})
+headers = {
+  'version': '2022-05-26',
```

Comment:
2022-04-07 (it will be easier to update the version later if it is the same date)

Reply:
Done